src/config/dynamic-theme-fixes.config

```diff
@@ -5141,6 +5141,9 @@ IGNORE IMAGE ANALYSIS
 
 reddit.com
 
+INVERT
```

Comment:
This will invert the logo, however this will interfere when you don't have the native dark mode enabled along side dark reader ![](https://i.imgur.com/jUgA9lI.png)

Reply:
Interesting, I don't seem to have this issue on my end.

Dark Reader + Reddit Dark Mode Disabled
![dark_disabled](https://user-images.githubusercontent.com/46883293/96187797-3e54de80-0f03-11eb-90ac-eab975af4ce6.png)

Dark Reader + Reddit Dark Mode Enabled
![dark_enabled](https://user-images.githubusercontent.com/46883293/96187806-40b73880-0f03-11eb-82dc-cee175958941.png)

Reply:
Hmm, weird ![](https://i.imgur.com/VZMM6fd.png).
Now I got it inverted back to almost non-visible in light mode.
 The result you got shouldn't be possible. As the dark/light mode of reddit changes the colors of the logo.

Reply:
Yeah, this is really strange. I can't seem to replicate what you're seeing, even in Firefox.
![firefox_2020-10-15_16 56 26](https://user-images.githubusercontent.com/46883293/96190196-72320300-0f07-11eb-9044-ec4fa02c4094.png)

Reply:
I can't even put my finger on why this is happening. Maybe going to this fix helps?  https://github.com/darkreader/darkreader/pull/3859#discussion_r505350067

Reply:
I tried that first, but it only worked when Reddit dark mode was enabled.

Reply:
Maybe add another CSS to correct the color so dark reader will first not interfere(ignore inline style) and CSS to force it it's original color in dark mode.